docs/static/releasenotes.asciidoc

```diff
@@ -26,6 +27,65 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+
+[[logstash-7-6-2]]
+=== Logstash 7.6.2 Release Notes
+
+
+* Fixed: Support for quoted plugin option key. The Java execution engine had a regression where adding quotes
+around plugin configuration keys would stop the pipeline from starting. https://github.com/elastic/logstash/pull/11694[#11694]
+* Fixed: Issue where users were not able to start pipeline when a configuration file was completely commented out. https://github.com/elastic/logstash/pull/11615[#11615]
+* Fixed: Typo in gauge metric of unknown type log. https://github.com/elastic/logstash/pull/11689[#11689]
+* Fixed: Issue where using command line `--help` option was showing wrong information. https://github.com/elastic/logstash/pull/11634[#11634]
+* [Doc] Backport more references to contributing issues guidelines. https://github.com/elastic/logstash/pull/11666[#11666]
+* [Doc] Add tips for troubleshooting a pipeline. https://github.com/elastic/logstash/pull/11545[#11545]
+* [Doc] Update to include verification mode switch. https://github.com/elastic/logstash/pull/11284[#11284]
+* [Doc] Updates security API examples https://github.com/elastic/logstash/pull/10752[#10752]
```

Comment:
Let's remove this one. The title is somewhat misleading.  It replaces the example with a link to the Elasticsearch Reference.